panels/examples/complexbin/animal.go

```diff
@@ -0,0 +1,110 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/vimeo/dials/panels"
+)
+
+var _ = panels.Must(p.Register("animals", defaultAnimals()))
```

Comment:
nit: let's move this registration to the bottom of the file.
... this way we're not referencing some thing that hasn't been created yet.